src/kernels/jupyter/launcher/jupyterExecution.ts

```diff
@@ -234,12 +234,12 @@ export class JupyterExecutionBase implements IJupyterExecution {
         if (options.localJupyter) {
             // If that works, then attempt to start the server
             traceInfo(`Launching server`);
-            const useDefaultConfig = !options || options.skipUsingDefaultConfig ? false : true;
```

Comment:
This value is really useless as all we do is read the settings and pass it down.
But when its used at that same place we already access the settings.
Hence removed this so that we can access the settings where this is required (no additional dependencies were injected as part of this refacor)